### PR TITLE
fix codesmell in callWithCbOrNextTick and closeWithCbOrNextTick

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: boot.js [hunk 1]
```diff
@@ -402,8 +402,8 @@ function thenify () {
   }
 }
 
-function callWithCbOrNextTick (func, cb, context) {
-  context = this._server
+function callWithCbOrNextTick (func, cb) {
+  const context = this._server
```
Comment (mcollina) [inline boot.js:406]: this is never called with the 3rd argument?

File: boot.js [hunk 2]
```diff
@@ -467,8 +467,8 @@ function timeoutCall (func, rootErr, context, cb) {
   }
 }
 
-function closeWithCbOrNextTick (func, cb, context) {
-  context = this._server
+function closeWithCbOrNextTick (func, cb) {
```
Comment (mcollina) [inline boot.js:470]: this is never called with the 3rd argument?

File: boot.js (hunk 1 above)
Comment (Uzlopak) [reply boot.js:406]: never called with 3rd argument

File: boot.js (hunk 2 above)
Comment (Uzlopak) [reply boot.js:470]: never called with 3rd argument

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**